activesupport/lib/active_support/benchmark.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module Benchmark # :nodoc:
```

Comment:
hey @Earlopain i can do the PR, but is it possible to remove this `:nodoc` because there is actually a doc right below :D 

and i couldnt find it here https://edgeapi.rubyonrails.org/files/activesupport/lib/active_support/core_ext/benchmark_rb.html

thanks!